### PR TITLE
!: add Spanish translation, fix placeholder in description

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/!.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Sustituye con un comando basado en un número de línea encontrado con `history`:
+
+`!{{número}}`
+
+- Sustituye con un comando que se utilizó un número especificado de líneas atrás:
+
+`!-{{número}}`
+
+- Sustituye por el comando más reciente que empiece por `cadema`:
```
Comment (kbdharun) [inline pages.es/common/!.md:18]:
```suggestion
- Sustituye por el comando más reciente que empiece por cadema:
```
This shouldn't probably be marked as a command as it is a placeholder, will update it in other pages too.

File: pages.es/common/!.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Signo de exclamación
```
[the remark this answers is not in the document]
Comment (kant) [reply pages.es/common/!.md]: You got the clue! Doing github in an ipad in a remote part of this planet earth is calamity! Thanks for your support!

**Comment by kbdharun** [comment]
I will update other pages too here, so marking it as a draft for a couple of minutes.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**